.github/workflows/build.yml

```diff
@@ -159,3 +159,30 @@ jobs:
       - name: "Perform quality checks: pylint_srctest"
         run: |
           temp/bin/tox -e pylint_srctest
+
+  publish:
+    name: "PyPI Publish"
+    needs: ["test", "quality"]
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    runs-on: "ubuntu-latest"
+    environment:
+      name: pypi
+      url: https://pypi.org/p/pyscard
+    permissions:
+      id-token: write
+
+    steps:
+
+      - name: Download artifacts
+        uses: actions/download-artifact@634f93cb2916e3fdff6788551b99b062d0335ce0 # 5.0.0
+
+      - name: move wheels to dist/
+        run: |
+          mkdir -v dist
+          mv -v wheel-*/*.whl dist/
+          ls -al dist/
+
+      - name: Publish wheels to PyPI
+        uses: pypa/gh-action-pypi-publish@ed0c53931b1dc9bd32cbe73a98c7f6766f8a527e # v1.13.0
+        with:
```

Comment:
It looks like this is no more needed.

https://github.com/pypa/gh-action-pypi-publish?tab=readme-ov-file#generating-and-uploading-attestations
> Generating signed [digital attestations](https://peps.python.org/pep-0740/) for all the distribution files and uploading them all together is now **on by default** for all projects using Trusted Publishing.